src/Farmer/Arm/Dns.fs

```diff
@@ -0,0 +1,155 @@
+[<AutoOpen>]
+module Farmer.Arm.Dns
+
+open Farmer
+open Farmer.CoreTypes
+open Farmer.Dns
+
+let zones = ResourceType ("Microsoft.Network/dnszones", "2018-05-01")
+let aRecord = ResourceType ("Microsoft.Network/dnszones/A", "2018-05-01")
+let aaaaRecord = ResourceType ("Microsoft.Network/dnszones/AAAA", "2018-05-01")
+let cnameRecord = ResourceType ("Microsoft.Network/dnszones/CNAME", "2018-05-01")
+let txtRecord = ResourceType ("Microsoft.Network/dnszones/TXT", "2018-05-01")
+let mxRecord = ResourceType ("Microsoft.Network/dnszones/MX", "2018-05-01")
+let nsRecord = ResourceType ("Microsoft.Network/dnszones/NS", "2018-05-01")
+let soaRecord = ResourceType ("Microsoft.Network/dnszones/SOA", "2018-05-01")
+let srvRecord = ResourceType ("Microsoft.Network/dnszones/SRV", "2018-05-01")
+let ptrRecord = ResourceType ("Microsoft.Network/dnszones/PTR", "2018-05-01")
+
+type DnsRecordType with
+    member this.ResourceType =
+        match this with
+        | Unknown -> failwith "Not Implemented"
+        | CName -> cnameRecord
+        | A -> aRecord
+        | AAAA -> aaaaRecord
+        | NS -> nsRecord
+        | PTR -> ptrRecord
+        | TXT -> txtRecord
+        | MX -> mxRecord
+        // | Soa -> soaRecord
```

Comment:
Can we remove these commented out items?

Reply:
Done